demo/index.html

```diff
@@ -1,84 +1,84 @@
 <!--
-    @license
-    Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
-    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
-    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
-    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
-    Code distributed by Google as part of the polymer project is also
-    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE
+The complete set of authors may be found at http://polymer.github.io/AUTHORS
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS
 -->
 <!doctype html>
 <html>
 <head>
   <title>paper-progress</title>
-  
+
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
   <meta name="mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-capable" content="yes">
-  
-  <script src="../webcomponentsjs/webcomponents.js"></script>
-  
-  <link rel="import" href="paper-progress.html">
-  <link rel="import" href="../paper-button/paper-button.html">
-  
-  <style shim-shadowdom>
-  
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+  <link rel="import" href="../paper-progress.html">
+  <link rel="import" href="../../paper-button/paper-button.html">
+
+  <style is="custom-style">
+
     body {
       font-family: RobotoDraft, 'Helvetica Neue', Helvetica, Arial;
       margin: 0;
       padding: 24px;
     }
-    
+
     paper-progress {
       display: block;
       width: 100%;
       padding: 25px 0;
     }
-    
-    paper-progress.pink::shadow #activeProgress {
-      background-color: #e91e63;
+
+    paper-progress.pink {
+      --paper-progress-active-color: #e91e63;
```

Comment:
Yeah. I wasn't sure.  I ended up using``--google-*`` colors as those are closer to the values that it had. Can you check again? thanks